e2e/tests/e2e.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from '@playwright/test'
```

Comment:
e2e are very cool also because we can do http requests testing also the server side rendered HTML output!